core/src/main/resources/error/error-classes.json

```diff
@@ -632,5 +632,12 @@
       "Writing job aborted"
     ],
     "sqlState" : "40000"
+  },
+  "CANNOT_DECODE_URL" : {
```

Comment:
Error classes should be sorted by names in the file. Please, put this before `CANNOT_INFER_DATE`